src/clr-angular/forms/radio/radio-container.ts

```diff
@@ -48,7 +47,7 @@ export class ClrRadioContainer implements OnDestroy {
    */
   @Input()
   set clrInline(value: boolean | string) {
-    if (!isBoolean(value)) {
+    if (typeof value === 'string') {
```

Comment:
So it looks like it's here to check whether the value is true or false regardless of its boolean or string. But if the string value is just a random string like "cat", `this.inline` would become true. Is that expected a behavior here?

Reply:
It's the expected behavior, yes. Normally `"false"` evaluates to `true` too, we just want to make that one exception to simplify the API a tiny bit if people don't use a binding here.

Reply:
`clrInline=""` is basically `true` state. This is about handling how HTML attributes work and not JavaScript, plus handling an edge case. 

In HTML the presence of an attribute determines it's boolean state, but with Angular binding we want to handle the case where `[clrInline]="false"` or clrInline="false" to make it more fault tolerant for typos.